packages/mobile/src/home/NotificationBox.tsx

```diff
@@ -38,6 +38,13 @@ import { getContentForCurrentLang } from 'src/utils/contentTranslations'
 import Logger from 'src/utils/Logger'
 
 const TAG = 'NotificationBox'
+// Priority of static notifications
+const BACKUP_PRIORITY = 1000
+const VERIFICATION_PRIORITY = 100
+const INVITES_PRIORITY = 400
+const INCOMING_PAYMENT_REQUESTS_PRIORITY = 300
+const OUTGOING_PAYMENT_REQUESTS_PRIORITY = 200
+const CELO_EDUCATION_PRIORITY = 10
```

Comment:
Nothing to do now, but looks like we should move this to our remote config so we can change the order of static notifications. What do you think?

Reply:
Yeah, I thought about this too. To be honest I didn't do it because I didn't want to deal with all the boilerplate of storing these things in Redux. Maybe we can have a simpler interface where we read directly from `remoteConfig()` since that already does caching and default management and bypass redux altogether?

Reply:
I think we should definitely reduce the boilerplate, but keep them in Redux so components using them are re-renders when their value changes.